mula/scheduler/context/context.py

```diff
@@ -41,6 +42,77 @@ def __init__(self) -> None:
         with Path(self.config.log_cfg).open("rt", encoding="utf-8") as f:
             logging.config.dictConfig(json.load(f))
 
+        structlog.configure(
```

Comment:
You might want to do an if-else clause here to prevent calling `structlog.configure` twice when JSON logging is enabled, especially since they're two different configurations and `structlog.configure` partially update (older) configurations.